src/components/docs/DocLayout.tsx

```diff
@@ -18,18 +18,17 @@ interface DocLayoutProps {
 export const DocLayout: FC<DocLayoutProps> = ({ children, ...customMeta }) => {
   const router = useRouter();
 
-  const baseUrl = config.raydiantUrl;
+  const baseUrl = config.raydiantDevelopersUrl;
```

Comment:
It looks like we were using the wrong URL here. This means the following values were referencing the incorrect url

```html
<meta property="og:url" content={`${meta.url}${router.asPath}`} />
<link rel="canonical" href={`${meta.url}${router.asPath}`} />
```

Perhaps that explains some of the weirdness that Ben and Anthony are seeing.